lib/mixins/dir-mixin.js

```diff
@@ -1,13 +1,13 @@
 /**
+ * @license Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
  * @suppress {checkPrototypalTypes}
```

Comment:
`@license` must be the last jsdoc tag in a comment, as it consumes all subsequent text

Reply:
Done

Reply:
Well, putting the `@suppress` before or after the `@license` comment doesn't seem to work. Where should I put the `@suppress`?